src/v3/package.json

```diff
@@ -76,6 +77,7 @@
     "js-cookie": "^3.0.1",
     "lodash": "^4.17.21",
     "preact": "^10.7.3",
+    "react-html-parser": "^2.0.2",
```

Comment:
Do we still need to do extra sanitization of the input strings before passing them to this library? https://github.com/peternewnham/react-html-parser#security

Reply:
Good point, I questioned this as well, but initially I thought it wasn't necessary since the content is coming from our properties/translation file. But I can see if somehow that was compromised it could be an added layer of protection. What do you think, should I add the `dompurify` lib? It seems relatively small ~700KB

Reply:
*Update* 

I actually gave this a try and when I use the `sanitize` function on the content from the properties file, it actually remove the `</$1>` token from the string and eventually breaks the content from displaying properly :( 

I don't think its a terrible thing to not sanitize since the content is sort of trusted (from .properties)

Reply:
_Additional update_ 

Now that I switched libraries (react-html-parser to html-react-parser). It seems to be doing some sort of sanitization on its own as it also removes the `<$1>` tags. However, I have a workaround for that, but now that this is in place, I think I can sanitize the string before parsing it, and should not have an issue like before, should I go ahead and do that?

Reply:
_Final update_

I went ahead and added the dompurify package and utilized it in the parser hook :D